tests/scrolling_panning_zooming/test_zoom_menu_correlation.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from selenium.webdriver import Firefox
+from modules.browser_object import PanelUi
+from modules.page_object_generics import GenericPage
+from time import sleep
+
+
+@pytest.fixture()
+def test_case():
+    return "65064"
+
+
+TEST_PAGE = "https://en.wikipedia.org"
+
+
+def test_zoom_menu_correlation(driver: Firefox):
+    # Open a webpage to test zoom level
+
+    page = GenericPage(driver, url=TEST_PAGE)
+    sleep(1)
+
+    # Open the Firefox Menu panel and prepare to interact with zoom controls
+    panel = PanelUi(driver).open()
+    panel.open_panel_menu()
+
+    # Switch to Firefox's Chrome context for UI interaction
+    with driver.context(driver.CONTEXT_CHROME):
+        # Step 1: Zoom out twice to approximately 90%
+        panel.get_element("zoom-reduce").click()
+        sleep(2)
```

Comment:
You can skip the context switching if you do `panel.click_on("zoom-reduce")`, it's in the BasePage and automatically switches contexts.